code/modules/mob/living/simple_animal/vore/carp.dm

```diff
@@ -19,13 +20,13 @@
 	pixel_x = -16
 	pixel_y = -16
 
-/mob/living/simple_animal/hostile/carp/large/huge
+/mob/living/simple_animal/hostile/carp/large
 	icon = 'icons/mob/vore64x64.dmi'
+	vore_icons = 0 // NO VORE SPRITES
```

Comment:
The problem is the big carp DOES have vore sprites...

Reply:
It has sprites in the vore file but no belly states. Only normal, -dead, and -gib icons.

Reply:
Welp it does have them now heh. Made some for it.